src/ops/transformations.jl

```diff
@@ -381,13 +381,14 @@ Returns:
     with_op_name(name, "Transpose") do
         if perm === nothing
             r = range(constant(0), LinearAlgebra.rank(n)-1)
-            perm = reverse(r, [true])
+            perm = reverse(r, [0])
```

Comment:
We should update `reverse(::Tensor, dims)` 
to match Julia's `reverse(::Tensor; dims)`

Reply:
done